src/docs/docs-block.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import klassy from '../klassy';
+
+var cl = klassy('docs-block');
+
+export default (props) => {
+  return <section { ...cl() } >
+    <aside { ...cl('info') } >
+      <h3 { ...cl('header') }> { props.title }</h3>
+    </aside>
+    <div { ...cl('content') }>
+      { props.children }
```

Comment:
This is an insertion point for all children.
Children are inserted "as is" without filtering/modifications.
Holes will be a more complex example. 
And `content-box` will be totally crazy :)